bindings/python/test/trajectory/test_propagator.py

```diff
@@ -77,7 +77,7 @@ def propagator(propagator_default_inputs) -> Propagator:
     return Propagator(*propagator_default_inputs[:2])
 
 
-class TestPropagated:
+class TestPropagator:
```

Comment:
I think this was a typo, noticed it while trying to dig up the stack trace.

Reply:
Yup, seems like it